src/Avalonia.Controls/Notifications/NotificationCard.cs

```diff
@@ -55,6 +55,8 @@ public NotificationCard()
                             break;
                     }
                 });
+
+                CloseOnClickProperty.Changed.AddClassHandler<Button>(OnCloseOnClickPropertyChanged);
```

Comment:
This should be in `static` constructor otherwise each `NotificationCard` instance will add extra handlers.